shuffler/logic/NW Sea/Isle of Gust/Gust.logic

```diff
@@ -0,0 +1,122 @@
+area Gust:
+  room LowerIsland:
+    node BeforeCave:
+      door Boat
+      door ToCaveLeft
+      mail Mailbox
+      enemy MiniBlinLeft1
+      enemy MiniBlinLeft2
+      enemy MiniBlinLeft3
+      enemy MiniBlinLeft4
+      enemy MiniBlinRight1
+      enemy MiniBlinRight2
+      enemy MiniBlinRight3
+      enemy MiniBlinRight4
+      enemy MiniBlinNearBombable1
+      enemy MiniBlinNearBombable2
+      enemy MiniBlinNearBombable3
+      enemy MiniBlinNearBombable4
+      enemy MiniBlinNearCave1
+      enemy MiniBlinNearCave2
+      enemy MiniBlinNearCave3
+      enemy MiniBlinNearCave4
+      enemy MiniBlinNearCave5
+      enemy RedChuChu1
+      enemy RedChuChu2
+      enemy RedChuChu3
+      enemy RedChuChu4
+      enemy RedChuChu5
+      enemy RedChuChu6
+
+    node BombableWall:
+      door ToBombableGrotto
+
+    node DigSpotsStart:
+      chest DigSpot1
+      chest DigSpot2
+      chest DigSpot3
+
+    BeforeCave <-> DigSpotsStart: item Shovel
+    BeforeCave -> BombableWall: (item Bombs | item Bombchus)
+    BombableWall -> BeforeCave
+
+
+
+    node AfterCave:
+      door ToCaveRight
+      door ToTempleFront
+      door ToTempleFrontLedgeRight
+      enemy RedChuChuRight1
+      enemy RedChuChuRight2
+      enemy RedChuChuLeft1
+      enemy RedChuChuLeft2
+      enemy RedChuChuLeft3
+      enemy MiniBlin1
+      enemy MiniBlin2
+
+    AfterCave -> BeforeCave
+
```

Comment:
A dig spot is missing here. It's really easy to miss because it's covered by grass at first
![dee_ess_island__20772](https://user-images.githubusercontent.com/91105575/199638010-d9dee8fb-1e0e-4a9c-8deb-9d764cf8253b.png)

```suggestion

    node DigSpotAfterCave:
      chest DigSpot4
    
    AfterCave <-> DigSpotAfterCave: item Shovel
```